src/core/api/mcda.ts

```diff
@@ -0,0 +1,8 @@
+import { apiClient } from '~core/apiClientInstance';
+import type { AxisDTO } from '~features/bivariate_manager/types';
+
+export function getBivariateAxes(abortController?: AbortController) {
+  return apiClient.get<AxisDTO[]>('/axis', undefined, true, {
```

Comment:
Right now endpoint returns 204 No Body on production, so this PR shouldn't be merged until that's resolved